data/shared/citizen/scripting/v8/main.js

```diff
@@ -112,8 +112,8 @@ const EXT_LOCALFUNCREF = 11;
 	global.addEventListener = addEventListener;
 	global.on = addEventListener;
 
-	global.RegisterNetEvent = function (name) { netSafeEventNames.add(name); };
-	global.RegisterServerEvent = global.RegisterNetEvent;
```

Comment:
And these are for backwards compatibility w/ emscripten-generated cpp resources.

Reply:
I added them in my first commit...